config/deploy/deployment.yaml.tpl

```diff
@@ -46,6 +46,25 @@ spec:
         - name: tls
           mountPath: "/tmp/k8s-metrics-server/serving-certs/"
           readOnly: true
+        ports:
+        - containerPort: 8080
+        readinessProbe:
+          tcpSocket:
+            port: 8080
```

Comment:
```suggestion
            port: 8443
```

8443 - the port at which the proxy is listening (what we care about)
8080 - the destination port at which `manager` container is exposing the  metrics endpoint/server.